frontend/server/src/Controllers/User.php

```diff
@@ -1544,29 +1516,30 @@ public static function apiSelectCoderOfTheMonth(\OmegaUp\Request $r): array {
             );
         }
         \OmegaUp\Validators::validateStringNonEmpty($r['username'], 'username');
-
-        $currentDate = date('Y-m-d', $currentTimestamp);
-        $firstDayOfNextMonth = new \DateTime($currentDate);
-        $firstDayOfNextMonth->modify('first day of next month');
-        $dateToSelect = $firstDayOfNextMonth->format('Y-m-d');
         \OmegaUp\Validators::validateOptionalInEnum(
             $r['category'],
             'category',
             \OmegaUp\Controllers\User::ALLOWED_CODER_OF_THE_MONTH_CATEGORIES
         );
         $category = $r['category'] ?? 'all';
-        $codersOfTheMonth = \OmegaUp\DAO\CoderOfTheMonth::getByTime(
+
+        $currentDate = date('Y-m-d', $currentTimestamp);
+        $firstDayOfNextMonth = new \DateTime($currentDate);
+        $firstDayOfNextMonth->modify('first day of next month');
+        $dateToSelect = $firstDayOfNextMonth->format('Y-m-d');
+
+        $codersOfTheMonth = \OmegaUp\DAO\CoderOfTheMonth::getByTimeAndSelected(
             $dateToSelect,
+            false,
```

Comment:
aquí no debería ser `true`? lo que se busca es ver si alguien explícitamente ya eligió al coder del mes siguiente.

Reply:
Si es false por que con false se trae al coder que este seleccionado, ya que si fuera true se traeria a todos los coder que no han sido seleccionados. https://github.com/omegaup/omegaup/blob/528dad7aebbcf96ad2b683a199e7b4ce193de442/frontend/server/src/DAO/CoderOfTheMonth.php#L295-L304

Reply:
OH es _`autoselected`_, no _`selected`_. puedes cambiar esta línea a

```suggestion
            /*autoselected=*/false,
```

? así es claro a qué se refiere para evitar confusiones como esta en el futuro.

Reply:
si muy bien